tests/eas/load_tracking.py

```diff
@@ -537,6 +540,406 @@ def test_load_avg_behaviour(self, experiment, tasks):
+        :return the json file name
+        """
+        rtapp = RTA(test_env.target, workload_name,
+            calibration=test_env.calibration())
+        rtapp.conf(kind='profile',
+            params={'task{}'.format(i): task.get() for i, task in enumerate(tasks)},
+            run_dir=test_env.target.working_directory)
+        return rtapp.json
+
+    @classmethod
+    def get_wload(cls, json_file):
+        """
+        Get a specification for a rt-app workload from a json file
+        :param json_file: path to the file that contains the tasks description
+        :type json_file: str
+
+        Example of expected task description:
+        "task0": {
+            "loop": 1,
+            "phases": {
+                "p1": {
+                    "cpus": [4],
+                    "loop": 62,
+                    "run": 4000,
+                    "timer": {
+                        "period": 16000,
+                        "ref": "task0"
+                    }
+                }
+            }
+        }
+        """
+
+        return {
+            'migration': {
+                'type' : 'rt-app',
+                'conf' : {
+                    'class' : 'custom',
+                    'json' : json_file,
+                    'prefix' : 'mig_test',
+                },
+            }
+        }
+
+    def _get_phases_names(self, tasks):
+        """
+        Return the phases names of a workload
+        :param task: Tasks description
+        :type task: dict
+        :return a list of the names of the phases
+        """
+        task_name = tasks.keys()[0]
+        return tasks[task_name]['phases'].keys()
+
+    def _get_duration_s(self, tasks, phase):
+        """
+        Compute the duration of a given phase
+        :param task: Tasks description
+        :type task: dict
+
+        :param phase: phase for which the duration is computed
+        :type phase: str
+        :return the duration in second of the phase
+        """
+        task_name = tasks.keys()[0]
+        # Get the loop value for a phase
+        loop = tasks[task_name]['phases'][phase]['loop']
+        # Get the period value for a phase in us
+        period_us = tasks[task_name]['phases'][phase]['timer']['period']
+        return (loop * period_us) / 1e6
+
+    def _get_duty_cycle(self, task, phase):
+        """
+        Compute the duty clycle of a task during a given phase.
+        :param task: task for which the duty clycle is computed
+        :type task: dict
+
+        :param phase: phase for which the duty cycle is computed
+        :type phase: str
+        :return the duty cycle in percent of a task for a given phase
+        """
+        # Get the period value for a phase in us
+        period_us = task['phases'][phase]['timer']['period']
+        # Get the run time value for a phase in us
+        run_time_us = task['phases'][phase]['run']
+        return run_time_us * 100 / period_us
+
+    def _get_cpu(self, task, phase):
```

Comment:
`_get_cpus`?... we can get a list back, isn't it?

... and the next methods should be named maybe `_get_tasks_cpus`?

Reply:
Done